MiniDumper/CommandLineOptions.cs

```diff
@@ -28,6 +28,12 @@ class CommandLineOptions
         [Option('l', HelpText = "Display the debug logging of the process + diagnostics info from the minidumper.")]
         public bool Verbose { get; set; }
 
+        [Option('y', HelpText = "Memory commit threshold in MB at which to create a dump.")]
+        public uint? MemoryCommitThreshold { get; set; }
+        
+        [Option('z', HelpText = "Memory commit threshold in MB at which to create a dump.")]
```

Comment:
Please change 'z' to "minmem" and also the help text is wrong, should be: "Trigger when memory commit drops below specified MB value."